tox.ini

```diff
@@ -5,7 +5,6 @@ envlist =
     black
     isort
     mypy
-    py3{9,10,11,12}
```

Comment:
When I run tox I don't see pytest run at all. Did you want to add `py3` to the list instead, or am I missing something ?

Reply:
Great catch! I do need `py3` indeed. Fixed in the new revision.